examples/data/dataview-category-source-mode.html

```diff
@@ -0,0 +1,163 @@
+    <title>Dataview Category Source mode</title>
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.css"
+    />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://libs.cartocdn.com/airship-style/v2.3/airship.css"
+    />
+    <!-- <link rel="stylesheet" type="text/css" href="https://libs.cartocdn.com/airship-icons/v2.3/icons.css"/> -->
+
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+
+      #map {
+        width: 100vw;
+        height: 100vh;
+      }
+    </style>
+  </head>
+
+  <body class="as-app-body as-app">
+    <div class="as-content">
+      <main class="as-main">
+        <div class="as-map-area">
+          <!-- map -->
+          <div id="map"></div>
+
+          <!-- panel -->
+          <div
+            class="as-panel as-panel--vertical as-panel--top as-panel--right as-bg--ui-01"
+          >
+            <div
+              class="as-panel__element as-p--16 as-body"
+              style="max-width: fit-content; padding-bottom: 0px;"
+            >
+              <p id="errorContainer" class="as-color--error"></p>
+              <p>
+                <span class="as-caption">Column</span>
+                <input
+                  id="column"
+                  class="as-input"
+                  type="text"
+                  value="continent"
+                />
+              </p>
+              <p>
+                <span class="as-caption">Limit</span>
+                <input id="limit" class="as-input" type="number" value="10" />
+              </p>
+              <p>
+                <span class="as-caption">Operation</span>
+                <as-dropdown
+                  default-text="Operation"
+                  can-clear="false"
+                  selected-option="count"
+                ></as-dropdown>
+              </p>
+              <p>
+                <span class="as-caption">Operation column</span>
+                <input
+                  id="operationColumn"
+                  class="as-input"
+                  type="text"
+                  value="pop_est"
+                />
+              </p>
+              <button
+                class="as-btn as-btn--secondary"
+                onclick="applyChanges();"
+              >
+                APPLY
+              </button>
+            </div>
+            <div
+              class="as-panel__element as-p--16 as-body"
+              style="max-width: fit-content; padding-top: 0px;"
+            >
+              <as-category-widget class="as-p--16" heading="Cities">
+              </as-category-widget>
+            </div>
+          </div>
+        </div>
+      </main>
+    </div>
+
+    <script src="https://api.tiles.mapbox.com/mapbox-gl-js/v1.10.0/mapbox-gl.js"></script>
+    <script src="https://unpkg.com/deck.gl@8.1.7/dist.min.js"></script>
```

Comment:
We have already merged the upgrade to deck.gl alpha. Please upgrade your branch / examples to it